Move automations into their own directory; update README (#201)
WordPress/openverse: automations/README.md

# Automations

This directory contains automations for managing the Openverse repositories and Developer Experience.

## Requirements

- Node v14
  - This should be kept in sync between the GitHub Actions `node-version` and the `volta.node` key in the `package.json` file.
- Python 3.10
- [Just](https://github.com/casey/just) command runner

## Installing Dependencies

> Use `just install` in this directory to set up Python and JavaScript dependencies.

Reviewer: Since `just` automatically finds the `justfile` and runs with the `justfile` directory as PWD, we don't need to specify the folder name.
```suggestion
Run `just install` to set up Python and JavaScript dependencies.
```
